src/styles/stars.scss

```diff
@@ -0,0 +1,27 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+  width: $width;
```

Comment:
The variable `$width` is used here but is not defined in this file. Ensure that it is defined elsewhere in your stylesheets or replace it with a specific value.